core/parachain/validator/parachain_processor.hpp

```diff
@@ -267,6 +269,9 @@ namespace kagome::parachain {
     std::vector<network::BackedCandidate> getBackedCandidates(
         const RelayHash &relay_parent) override;
 
+    // CanDisconnect
```

Comment:
This is confusing out of PR context.

Reply:
No, this is comment documenting what interface group of `override`s implements